app/images_parser.py

```diff
@@ -30,9 +32,39 @@ def image_records():
             #"filename": img.filename,
             "uuid": img.uuid,
             #"ext": img.ext
+            #"leafiness": img.leafiness
         } for img in parse_images()
     ]
 
+def image_records_with_classifications():
```

Comment:
Move this method into a new file called "images_classifier.py". It should reach into "images_parser.py" to collect all the images and "image_classifier.py" to classify each.